src/ui/public/agg_response/tabify/_buckets.js

```diff
@@ -81,4 +84,23 @@ TabifyBuckets.prototype._orderBucketsAccordingToParams = function (params) {
   }
 };
 
+// dropPartials should only be called if the aggParam setting is enabled,
+// and the agg field is the same as the Time Range.
+TabifyBuckets.prototype._dropPartials = function (params, timeRange) {
+  if (!timeRange ||
+    this.buckets.length <= 1 ||
+    this.objectMode ||
+    params.field.name !== timeRange.name) return;
```

Comment:
nit: it's better to align with our styleguide always-use-braces-for-conditionals-and-loops